src/arrayMethodJoin.js

```diff
@@ -4,8 +4,20 @@
  * Implement method join
  */
 function applyCustomJoin() {
-  [].__proto__.join2 = function(separator) {
-    // write code here
+  [].__proto__.join2 = function(separator = ',') {
+    let copy = '';
```

Comment:
It is better to use a more descriptive name for the variable 'copy', like 'joinedString' for better code readability.